### PR TITLE
Accordion Animations

### PR DESCRIPTION
**Pull request by KarnellSchultz**
### [Easings.co](https://easings.co/)

Added open/close keyframe animations to the accordion.

Tried several ways to do this with some issues around the padding the `<Content />` component had. Decided to remove the padding since the block itself has padding. However, going forward I'd like to move the padding into the accordion rather than having it on the block.

## Justify why they are needed

[Before demo fixes](https://www.notion.so/hedviginsurance/Before-Demo-Fixes-0f8755117f6e4092b16ae0743dc7bcec?pvs=4)

## Checklist before requesting a review

- [ ] I have performed a self-review of my code

### CODE REVIEW

File: apps/store/src/components/Accordion/Accordion.tsx [hunk 1]
```diff
@@ -64,9 +65,34 @@ const CloseIcon = styled(MinusIcon)({
   '[data-state=open] &': { display: 'block' },
 })
 
+const slideDown = keyframes({
+  from: {
+    height: 0,
+  },
+  to: {
+    height: 'var(--radix-accordion-content-height)',
```
Comment (gustaveen) [inline apps/store/src/components/Accordion/Accordion.tsx:73]: Where is this variable defined?

**Review by gustaveen — APPROVED** [review]
Nice, very smoooth! 🙌

Maybe it can speed up a bit but I'll let Zak and Wille be the judge of that :D

File: apps/store/src/components/Accordion/Accordion.tsx (hunk 1 above)
Comment (KarnellSchultz) [reply apps/store/src/components/Accordion/Accordion.tsx:73]: it's coming from [radix](https://www.radix-ui.com/docs/primitives/components/accordion) 

Should I drop a comment on that?

**Comment by KarnellSchultz** [comment]
slack convo: https://hedviginsurance.slack.com/archives/C041SD67G82/p1675955904078479

**Pull request merged**